web/templates/settings/admin_tab.hbs

```diff
@@ -13,9 +13,11 @@
 
 {{> emoji_settings_admin }}
 
+{{#if is_admin}}
 {{> user_list_admin }}
-
-{{> deactivated_users_admin }}
+{{else}}
```

Comment:
I don't understand these changes. I was thinking of only having {{user_list_admin}} here and then user_list_admin.hbs include the three templates for three tabs for all type of users.

Reply:
This was done to ensure that for non-admins we don't need to show all the three tabs , the Users section would just be sufficient as it was before.

Reply:
Was this discussed anywhere? We currently show all three sections as different panels in left side to non-admins.

Reply:
Ohh my bad , didn't look at the "Show more" section. 
I have updated those changes.